### DISCUSSION TITLE
DOI of Journal Article

### DISCUSSION THREAD

**Discussion by Julianlee24** [Biol2299-Matthis-Fall2022/Biol2299-Matthis-Fall2022#33] [General]
Can an article be so outdated, the DOI no longer works???

**Comment by andreabuitrago1** [comment]
probably

**Comment by glor-n** [comment]
no cause they dont like expire maybe it was just user error :)

**Comment by jonmatthis** [comment]
Although I did reply "probably" when you asked this question in the classroom (which @andreabuitrago1 dutifully copied here), I actually kinda think @glor-n is right? 

like, if you found a DOI on a paper, then it *should* still work. Maybe a typo? or did you miss some of the characters?

**Comment by Julianlee24** [comment]
I sure hope it is not due to user error, but ive tried multiple sites and the doi always opens an error page.
